buildroot-external/board/pc/ova/kernel.config

```diff
@@ -24,6 +31,10 @@ CONFIG_VIRTIO_VSOCKETS=m
 CONFIG_VIRTIO_MMIO=y
 CONFIG_SCSI_VIRTIO=y
 CONFIG_HW_RANDOM_VIRTIO=y
+CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
```

Comment:
I don't think that this is really necessary/required right now, but it might come in useful in the future or for testing.

Reply:
Agreed.